docs/tutorials/how-to-sampler.ipynb

```diff
@@ -0,0 +1,132 @@
+    "    - **circuit_indices**:\n",
+    "        - **Type**: array\n",
+    "        - **Description**: Indexes of the circuits to evaluate.\n",
+    "        - **Required**: True\n",
+    "    - **parameter_values**\n",
+    "        - **Type**: array\n",
+    "        - **Description**: Concrete parameters to be bound.\n",
+    "        - **Required**: False\n",
+    "- **Input parameters for Sampler**:\n",
+    "    - **skip_transpilation**:\n",
+    "        - **Description**: Skip circuit transpilation. The default value is False.\n",
+    "        - **Required**: False.   \n",
+    "    - **circuits**:\n",
+    "        - **Description**: A QuantumCircuit or list of QuantumCircuits. The circuits can be parameterized.\n",
+    "        - **Required**: True\n",
+    "    - **parameters**\n",
+    "        - **Type**: array\n",
+    "        - **Description**: Parameters of the quantum circuits.\n",
+    "        - **Required**: False       \n",
+    "    - **options**        \n",
+    "        - **Description**: Additional parameters, such as the backend to use.\n",
+    "        - **Required**: False\n",
+    "- **Returns**:\n",
+    "   - **metadata**:\n",
+    "        - **Type**: array\n",
+    "        - **Description**: Additional metadata.  \n",
+    "        - **Required**: False\n",
+    "   - **quasi_dists**:\n",
+    "       - **Description**: Quasidistribution for each circuit.\n",
+    "       - **Type**: array\n",
+    "       - **Required**: False\n",
+    "\n",
+    "## Example\n",
+    "{: #program-sampler-sample-code}\n",
+    "\n",
+    "```Python\n",
+    "from qiskit_ibm_runtime import QiskitRuntimeService, Sampler\n",
+    "from qiskit import QuantumCircuit\n",
+    "\n",
+    "service = QiskitRuntimeService(channel=\"ibm_cloud\", token=\"<api-token>\", instance=\"<IBM Cloud CRN>\")\n",
+    "\n",
+    "bell = QuantumCircuit(2)\n",
+    "bell.h(0)\n",
+    "bell.cx(0, 1)\n",
+    "bell.measure_all()\n",
+    "\n",
+    "# executes a Bell circuit\n",
+    "with Sampler(circuits=bell, service=service, options={ \"backend\": \"ibmq_qasm_simulator\" }) as sampler:\n",
+    "    result = sampler(circuit_indices=[0], shots=1024)\n",
+    "    print(result)\n",
+    "```\n",
+    "{: codeblock}\n",
+    "\n",
+    "```text\n",
+    "SamplerResult(quasi_dists=[{'00': 0.4873046875, '11': 0.5126953125}], metadata=[{'header_metadata': None, 'shots': 1024}])\n",
+    "```\n",
+    "\n",
+    "## Next steps\n",
+    "{: #next-steps}\n",
+    "\n",
+    "- Use the [Getting started guide](/docs/quantum-computing?topic=quantum-computing-quickstart) to create an instance and run your first job.\n",
+    "- Review [Get started with the Sampler primitive](/docs/quantum-computing?topic=quantum-computing-example-sampler) for a step-by-step example of using this primitive.\n",
+    "- Use Qiskit [tutorials](https://qiskit.org/documentation/tutorials.html){: external} to learn how to create circuits with Qiskit.\n"
```

Comment:
I would suggest we just remove this section for now and add one when we have other pages.